thirdeye/thirdeye-pinot/src/main/resources/assets/js/lib/tabular.js

```diff
@@ -169,7 +169,7 @@ function drawTimeSeries(timeSeriesData, tab) {
             y: {
                 tick: {
                     //format integers with comma-grouping for thousands
-                    format: d3.format(',.0f')
+                    format: d3.format(",.1 ")
```

Comment:
Space is needed it means: either decimal or exponent notation, removing insignificant trailing 
zeros.